src/main/java/com/google/sps/servlets/ListOrganizationsServlet.java

```diff
@@ -81,14 +81,23 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     QueryResultList<Entity> results = prepQuery.asQueryResultList(fetchOptions);
     ArrayList<Organization> requestedOrganizations = new ArrayList<Organization>();
 
-    /* Fills requestedOrganizations array with 4 fields of each org- name, phone, addres, and desc. */
+    /* Fills requestedOrganizations array*/
     for (Entity entity : results) {
-
-      Organization newOrg = new Organization((String) entity.getProperty("orgName"),
-                                             (String) entity.getProperty("orgEmail"),
+      long openingHour = (long) ((ArrayList)entity.getProperty("openHours")).get(0);
+      long closingHour = (long) ((ArrayList)entity.getProperty("openHours")).get(1);
+      Organization newOrg = new Organization((long) entity.getKey().getId(), 
```

Comment:
This function is getting quite long. How would you start to factor it apart into reusable components?

What do you think of moving this functionality to the Organizations class?